db/migrate/20200907114036_create_computacenter_user_ledger.rb

```diff
@@ -0,0 +1,25 @@
+class CreateComputacenterUserLedger < ActiveRecord::Migration[6.0]
+  def change
+    create_table :computacenter_user_ledgers do |t|
+      t.integer :user_id
+      t.text :first_name
+      t.text :last_name
+      t.text :email
+      t.text :telephone
+      t.text :responsible_body
+      t.text :responsible_body_urn
+      t.text :cc_sold_to_number
+      t.text :school
+      t.text :school_urn
+      t.text :cc_ship_to_number
+      t.datetime :updated_at_timestamp
+      t.text :type_of_update
+      t.text :original_email
```

Comment:
Can we please change this for consistency?

```suggestion
      t.text :original_email_address
```

Reply:
sure, maybe we can alias everything to `email` in the near future?